stellargraph/core/graph.py

```diff
@@ -906,18 +930,28 @@ def str_node_type(count, nt):
             sep="\n  ",
         )
 
-        # FIXME: it would be better for the schema to just include the counts directly
-        unique_ets = self._unique_type_triples(return_counts=True)
+        metric_names = ["count", "min", "max", "mean", "std"]
+        et_metrics = self._edge_metrics_by_type_triple(metrics=metric_names)
+
         edge_types = sorted(
             (
-                (count, EdgeType(src_ty, rel_ty, tgt_ty))
-                for src_ty, rel_ty, tgt_ty, count in unique_ets
+                (
+                    metrics["count"],
+                    EdgeType(src_ty, rel_ty, tgt_ty),
+                    _WeightInfo(
```

Comment:
This named tuple is defined a long way away, so it's not obvious what it stores. I think it'd be ok to either define it within this function so that the connection is stronger, or, since all of this processing is tightly scoped, not define it at all by just passing around the `metrics` value.

```python
(metrics["count"], EdgeType(src_ty, rel_ty, tgt_ty), metrics)
```

What do you think?

Reply:
Yeah I think just passing around `metrics` is probably fine here, it's only being used in this function 👍 I'll update